core/backend/src/TxnManager.ts

```diff
@@ -111,11 +112,8 @@ class ChangedEntitiesProc implements TxnChangedEntities {
       });
 
       changes.sendEvent(iModel, changedEvent, evtName);
-    } catch (_) {
-      // Presumably, the temp txn tables don't exist, because the native TxnManager is not tracking changes.
-      // This occurs when the application is using a "pull-only" briefcase - they open the briefcase as read-write temporarily,

```

Comment:
Comment explaining this code deleted - why?

Reply:
That comment is no longer true. If we ever get this exception it is unexpected.